resources/assets/components/SiteNavigation/SiteNavigation.js

```diff
@@ -511,20 +511,132 @@ class SiteNavigation extends React.Component {
+              onMouseEnter={() => this.handleMouseEnter('BenefitsSubNav')}
+              onMouseLeave={() => this.handleMouseLeave('BenefitsSubNav')}
+            >
+              <Media
+                queries={{
+                  large: '(min-width: 960px)',
+                }}
               >
-                Scholarships
-              </a>
+                {matches => (
+                  <>
+                    {matches.large ? (
+                      <a
+                        id="main-nav__benefits"
+                        href="/us/about/benefits"
+                        onClick={() =>
+                          this.handleOnClickLink({
+                            name: 'clicked_nav_link_benefits',
+                            action: 'link_clicked',
+                            category: EVENT_CATEGORIES.navigation,
+                            label: 'benefits',
+                          })
+                        }
+                      >
+                        Benefits
+                        <span className="main-nav__arrow" />
+                      </a>
+                    ) : (
+                      <a
+                        id="main-nav__benefits"
+                        href="/"
+                        onClick={e =>
+                          this.handleOnClickToggle(e, 'BenefitsSubNav', {
+                            name: 'clicked_nav_link_benefits',
+                            action: 'link_clicked',
+                            category: EVENT_CATEGORIES.navigation,
+                            label: 'benefits',
+                          })
+                        }
+                      >
+                        Benefits
+                        <span className="main-nav__arrow" />
+                      </a>
+                    )}
+                  </>
+                )}
+              </Media>
+
+              {this.state.activeSubNav === 'BenefitsSubNav' ? (
+                <div className="main-subnav menu-subnav">
+                  <div className="wrapper base-12-grid py-3 md:py-6">
+                    <section className="main-subnav__links-benefits menu-subnav__links menu-subnav__section">
+                      <h1>Benefits</h1>
+                      <ul>
+                        <li>
+                          <a
+                            href="/us/about/easy-scholarships"
+                            onClick={() => {
+                              this.handleOnClickLink({
+                                name:
+                                  'clicked_subnav_link_benefits_scholarships',
+                                action: 'link_clicked',
+                                category: EVENT_CATEGORIES.navigation,
+                                label: 'benefits',
+                              });
+                            }}
+                          >
+                            Scholarships
+                          </a>
+                          <a
```

Comment:
These `<a>`'s shouldn't be siblings; they should each be in their own `<li>`s, so:

```suggestion
                              </a>
                          </li>
                          <li>
                              <a
```